### ISSUE TITLE
Cancel button

### ISSUE THREAD

**Issue by enguerranws** [gs-akhan/react-native-chooser#40]
Hi, 

I'm just wondering how to make a cancel button (or cancel on a tap on the backdrop) using this component. Any ideas?

**Comment by gs-akhan** [comment]
Hi,
Tap on background should close the dropdown.

**Comment by enguerranws** [comment]
Hi @gs-akhan, 

sure it does, but adding a close button (a simple cross in the corner) should make this less implicit? That being said, I guess I can do it by simply adding some styles.

**Comment by enguerranws** [comment]
> That being said, I guess I can do it by simply adding some styles.

Well, no. I would simply add an `<Image/>` component on the backdrop, but it can't be done right now. Another option would be to have a `onOpen` `onClose` property so I could display / hide an 
external component based on a state property.

**Comment by gs-akhan** [comment]
The close button makes sense. You can add it.

Btw how do you plan to add that cross, as an image ? SVG ?

Thanks

**Comment by enguerranws** [comment]
> Btw how do you plan to add that cross, as an image ? SVG ?

Right now, I think of a RN Image component (as SVG support means some react-native svg module dependency). I actually don't know how to achieve that right now using your plugin: is it possible?

**Comment by gs-akhan** [comment]
There is an indicator component that is used to render a drop icon . You
can check it .. in this very library

**Comment by enguerranws** [comment]
>There is an indicator component that is used to render a drop icon

The indicator property shows an indicator on the select item itself, not in the modal. I would like to find something to add a close / cancel button inside the modal backdrop.

**Comment by gs-akhan** [comment]
Yes, I was suggesting you can checkout how that drop icon was built. Or you can use ImageComponent too. I am fine.
